soccer/SimFieldView.cpp

```diff
@@ -98,9 +101,13 @@ void SimFieldView::mouseMoveEvent(QMouseEvent* me) {
 
 void SimFieldView::mouseReleaseEvent(QMouseEvent* me) {
     if (_dragMode == DRAG_SHOOT) {
-        SimCommand cmd;
-        *cmd.mutable_ball_vel() = _teamToWorld.transformDirection(_shot);
-        sendSimCommand(cmd);
+        grSim_Packet simPacket;
```

Comment:
Dragging and letting go of the ball seems to be crashing soccer for me: 
```
[libprotobuf FATAL /var/tmp/portage/dev-libs/protobuf-3.1.0/work/protobuf-3.1.0/src/google/protobuf/message_lite.cc:289] CHECK failed: IsInitialized(): Can't serialize message of type "grSim_Packet" because it is missing required fields: replacement.ball.x, replacement.ball.y
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():  CHECK failed: IsInitialized(): Can't serialize message of type "grSim_Packet" because it is missing required fields: replacement.ball.x, replacement.ball.y
make: *** [makefile:32: run-sim] Aborted
```